lua/unreal/commands.lua

```diff
@@ -656,6 +642,38 @@ function Commands.GetProjectName()
     return CurrentGenData.prjName .. ".uproject"
 end
 
+function WindowsPathBuilder(CurrentGenData)
+    CurrentGenData.ubtPath = "\"" .. CurrentGenData.config.EngineDir .."/Engine/Binaries/DotNET/UnrealBuildTool/UnrealBuildTool.exe\""
+    CurrentGenData.ueBuildBat = "\"" .. CurrentGenData.config.EngineDir .."/Engine/Build/BatchFiles/Build.bat\""
+    CurrentGenData.projectPath = "\"" .. CurrentGenData.prjDir .. "/" ..
+        CurrentGenData.prjName .. ".uproject\""
+end
+
+function LinuxPathBuilder(CurrentGenData)
+    CurrentGenData.ubtPath =   CurrentGenData.config.EngineDir .."/Engine/Binaries/ThirdParty/DotNet/6.0.302/linux/dotnet "..
```

Comment:
hardcoded path to specific dotnet version. I understand this is shipped with the engine, however looks prone to easy breakage as soon as the engine switches the version, is there a better way to implement this perhaps?